bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/InstrumentDetails.scala

```diff
@@ -82,3 +82,28 @@ final case class GemsParameters(
                      avgStrehl:           Double,
                      strehlBand:          String)
 
+
+object InstrumentDetails {
+
+  // NOTE: This is similar to the code in ItcUniqueConfig which decides on imaging or spectroscopy setup
+  // on Config elements. There should be a way to share this?
```

Comment:
I suppose we could define an `ObservingMode` (or something) type class that provides the imaging/spectroscopy distinction.  It would still be implemented differently for all the different kinds of instrument/config representations but at least it would be easier to track them down.